packages/hub/src/start-dev-server.ts

```diff
@@ -1,3 +1,5 @@
+import '../config/env'; // Note: importing this module has the side effect of modifying env vars
```

Comment:
I wonder if it would be clearer to just import and run `configureEnvVariables()` here. That's all that `../config/env` does, and it seems less magic.

Reply:
I thought about that as well. `env.js` is also used for setting env vars for [jest unit tests](https://github.com/statechannels/monorepo/blob/master/packages/hub/config/jest/jest.config.js#L7) and for setting up ganache for [unit tests that require ganache](https://github.com/statechannels/monorepo/blob/master/packages/hub/jest/test-setup.ts#L1).

My thinking was that it could be worthwhile to keep a module for setting env vars in case the module evolves to more than just the call to `configureEnvVariables()`. That being said, I don't feel strongly about it.